grunt/browserify.js

```diff
@@ -29,5 +29,12 @@ module.exports = {
 		},
 		src: 'node_modules/ecurve/lib/index.js',
 		dest: 'build/ecurve.js'
+	},
+	querystring: {
```

Comment:
Why `querystring` ? Is that not meant to be used for Bitcoin?

Reply:
There are edge cases when dealing with query strings. No need to re-invent what is already solved.